templates/kubernetes/install.html

```diff
@@ -477,14 +523,14 @@ <h2>Kubernetes resources</h2>
           <p class="p-heading-icon__title p-muted-heading" style="color: #fff;">Data Sheet</p>
         </div>
       </div>
-      <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
+      <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise&nbsp;&rsaquo;</a></p>
```

Comment:
This change is not specified in the copy doc

Reply:
Chevron usage is not decided on the copydoc level, but on the design level. usually there is some consistency of use within a page, so In this case I was trying to get all the links to align with one another